src/components/molecules/TableGrid/TableGrid.tsx

```diff
@@ -38,21 +39,62 @@ export interface TablesUserListProps {
   leaveText?: string;
   venue: WithId<AnyVenue>;
   venueId: string;
-  template: VenueTemplate;
+  userId: string;
 }
 
-export const TablesUserList: React.FC<TablesUserListProps> = ({
+export const TableGrid: React.FC<TableGridProps> = ({
   venueId,
-  setSeatedAtTable,
-  seatedAtTable,
   customTables,
   defaultTables,
   showOnlyAvailableTables = false,
   TableComponent,
   joinMessage,
   venue,
```

Comment:
About time to rename to `space` and use `spaceId` inside this component and in the one or two places its props are passed?